src/parachain/vaults.ts

```diff
@@ -435,14 +434,27 @@ export class DefaultVaultsAPI extends DefaultTransactionAPI implements VaultsAPI
         return new Big(satToBTC(vault.issued_tokens.toString()));
     }
 
-    private async getIssuedPolkaBTCAmounts(): Promise<Big[]> {
+    async getIssuableAmount(vaultId: AccountId): Promise<Big> {
```

Comment:
This function should be added to the interface so the UI can access it

Reply:
I think it already is right? Or is there another interface I missed?